# What: Allow current_cpu=riscv64 to pass the check to use 64 bits
# Author: julianmorillo (BSC)
--- qtwebengine/src/3rdparty/chromium/third_party/angle/gni/angle.gni.orig	2025-02-03 17:36:58.966944504 +0100
+++ qtwebengine/src/3rdparty/chromium/third_party/angle/gni/angle.gni	2025-02-03 17:42:15.602107628 +0100
@@ -104,7 +104,7 @@
 
   if (current_cpu == "arm64" || current_cpu == "x64" ||
       current_cpu == "mips64el" || current_cpu == "s390x" ||
-      current_cpu == "ppc64" || current_cpu == "loong64") {
+      current_cpu == "ppc64" || current_cpu == "loong64" || current_cpu == "riscv64") {
     angle_64bit_current_cpu = true
   } else if (current_cpu == "arm" || current_cpu == "x86" ||
              current_cpu == "mipsel" || current_cpu == "s390" ||
